raiden-ts/docs-source/SDK-Development.md

```diff
@@ -118,6 +118,45 @@ The project is structured in a domain-driven logic, each folder under `src` repr
 
 These are just suggestions on how to keep a well organized codebase. It is the developer's responsibility to decide in which module/domain any function, data or type belongs.
 
+## Data Persistence
+
+The visual representation of the data handling and persistance:
+
+```
+                         +--------------------+
+                         |       State        |
+          .............. |      [Redux]       | <.............
+          .              |    (in memory)     |              .
+          .              +--------------------+              .
+          .                        ^                         .
+          .                        .                         .
+   async sync back         load data on start        read historic data
+          .                        .                         .
+          .                        .                         .
+          .              +--------------------+              .
+          .              |     Database       |              .
+          .............> |     [PouchDB]      | ..............
+                         |(persistent storage)|
+                         +--------------------+
+                                   ^
+                                   |
+          +------------------------+------------------------+
+          |                        |                        |
++--------------------+   +--------------------+   +--------------------+
+|  LevelDB Adapter   |   | IndexedDB Adapter  |   |   Memory Adapter   |
+|     (NodeJS)       |   |   (Web-Browser)    |   |      (Tests)       |
++--------------------+   +--------------------+   +--------------------+
+```
+
+Additional notes:
+
+- The database adapter gets chosen based on the environment the SDK is running inside.
+- The hot state in memory with Redux is limiting its size by not including historic data of transfers and old channels.
```

Comment:
(our) old channels are kept in the memory state, since these shouldn't ever grow much. We could eventually move them out, but for now they stay. Only the historic (completed) transfers are cleaned by a specific `transfer/clear` action.

Reply:
:heavy_check_mark: